src/content/docs/mobile-monitoring/new-relic-mobile/maintenance/configure-settings-for-mobile-monitoring.mdx

```diff
@@ -112,16 +112,15 @@ To configure these settings, go to **[one.newrelic.com > All capabilities](https
 
 ## View your application token [#application-token]
 
-One of our [data ingest keys](/docs/apis/intro-apis/new-relic-api-keys) is our mobile monitoring application token. For each mobile app you monitor, we automatically assign a unique, 40-character hexadecimal string to that app, for both the US and EU datacenter. This token is required to authenticate your app's data.
+Your mobile app token is a unique, 40-character hexadecimal string that authenticates your app's data to New Relic. Each mobile app you monitor has a unique token for both the US and EU datacenters.
+
+All data from your mobile app is reported to the same place in the New Relic mobile monitoring UI, regardless of the app's name when installed on a mobile device. If you're using a hybrid framework such as Flutter or React Native and deploying to both iOS and Android platforms, you'll need to use separate tokens, one for iOS and one for Android.
```

Comment:
Colon, I think the second sentence should be broken up more. Also in the first sentence you can change "is reported" to "reports" for a quick passive voice fix, I think.